airbyte-ci/connectors/pipelines/pipelines/actions/environments/os/alpine.py

```diff
@@ -0,0 +1,50 @@
+from typing import List
+from pipelines.utils import sh_dash_c
+
+
+from dagger import Container
+
+
+def with_git(dagger_client, ci_github_access_token_secret, ci_git_user) -> Container:
```

Comment:
I've been more recently inclined in declaring not re-used function building container in the context they're used. Following this philisophy, `with_git` would be declared in the `git.py` module.